### PR TITLE
Enhacement/add unit tests

### PR DESCRIPTION
**Pull request by pgarciaegido**
Hey, 

In this PR I include Jest + Testing library to be able to create unit tests and execute them with `yarn test`.

Also, I could add a github action for them to run on PR. WDYT?

### CODE REVIEW

**Comment by angel-paella** [comment]
Can we make sure these execute automatically? It can be either in PRs or in build, but either we make sure they execute or we risk basically forgetting about them

**Comment by pgarciaegido** [comment]
@angel-paella I just edited my description haha. Yes, Ill create a github action on PR this evening👍

File: packages/ui/react-ui/__tests__/CrossmintPayButton.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, fireEvent, screen, act } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import { CrossmintPayButton } from "../src/CrossmintPayButton";
+import { LIB_VERSION } from "../src/version";
+
+// Todo: create a global service for this to work everywhere and to be able to customize resolved/rejected responses
```
Comment (alfonso-paella) [inline packages/ui/react-ui/__tests__/CrossmintPayButton.spec.tsx:8]: 1. Can you name this as "TODO" ?

We are trying to standardize the way we write it so it's easy to set automations and rules around it

2. Can you file an issue on GH and attach the issue number here?

Example:

`// TODO(#777): create a global service for this to work everywhere and to be able to customize resolved/rejected responses`

**Review by alfonso-paella — APPROVED** [review]
Approved with comments

**Pull request merged**